### PR TITLE
Update release checklist

### PR DESCRIPTION
**Pull request by nikola-matic**

### CODE REVIEW

File: ReleaseChecklist.md [hunk 1]
```diff
@@ -39,6 +39,7 @@ At least a day before the release:
 
 ### Blog Post
  - [ ] Create a post on [solidity-website](https://github.com/ethereum/solidity-website/tree/main/src/posts) in the ``Releases`` category and explain some of the new features or concepts.
+       **Note**: This should include the update of ``index.tsx`` to update the release version and description blurb.
```
Comment (r0qs) [inline ReleaseChecklist.md]:
```suggestion
       **Note**: This should include the update of [index.tsx](https://github.com/ethereum/solidity-website/blob/main/src/pages/index.tsx) with the new release version and description in the Solidity website main page.
```

**Review by r0qs — APPROVED** [review]

File: ReleaseChecklist.md [hunk 2]
```diff
@@ -39,6 +39,7 @@ At least a day before the release:
 
 ### Blog Post
  - [ ] Create a post on [solidity-website](https://github.com/ethereum/solidity-website/tree/main/src/posts) in the ``Releases`` category and explain some of the new features or concepts.
+       **Note**: This should include the update of [index.tsx](https://github.com/ethereum/solidity-website/blob/main/src/pages/index.tsx) with the new release version and description in the Solidity website main page.
```
Comment (cameel) [inline ReleaseChecklist.md:42]: I always assumed it was covered by this:

> Update the release information section [in the source of soliditylang.org](https://github.com/ethereum/solidity-website/blob/main/src/pages/index.tsx).

Maybe we should just reword it to make it clearer?

Comment (nikola-matic) [reply ReleaseChecklist.md:42]: You're right, I'll close this one.
